src/Components/EmailVerificationPage.jsx

```diff
@@ -0,0 +1,23 @@
+import "./emailVerificationPage.scss";
+
+export default function EmailVerificationPage() {
+  const verifyEmailHandler = async () => {
+    const verification = await authService.createUserVerification();
```

Comment:
we also need to check if the user is already verified or not bcoz if they are then we will not call createVerification api instead we will just navigate them to the home page